js/job.js

```diff
@@ -8,40 +8,19 @@ function Job (opt) {
   this.responsibilities = opt.responsibilities;
 }
 
-Job.prototype.toHtml = function() {
-  var source = $('#job-template').html();
-  var template = Handlebars.compile(source);
-  return template(this);
-};
-$.getJSON('data/jobHistory.json', function(data){
-  localStorage.jobHistory = JSON.stringify(data);
-  data.forEach(function(ele) {
-    jobs.push(new Job(ele));
+var retrieveJobHistory = function(){
+  $.getJSON('https://raw.githubusercontent.com/tetsaiga86/Portfolio/class-07/data/jobHistory.json', function(data){
+    localStorage.jobHistory = JSON.stringify(data);
+    data.forEach(function(ele) {
```

Comment:
Hey, you're getting a collection here (JSON array) and trying to turn it into a different collection (array of Jobs)....what array method can we use here?